[BUG] make reviewable fails

According to the development guid
<img width="1250" alt="image" src="https://github.com/apecloud/kubeblocks/assets/111829739/80df9d91-8899-4f5a-a650-302784acc940">

One should run `make reviewable` and make sure it success.

But on the latest main branch, it fails:
<img width="760" alt="image" src="https://github.com/apecloud/kubeblocks/assets/111829739/d1d7e825-0c48-4457-bcc7-15efb702fdfe">

### Append `test-go-generate` to the dependency list of the target `vet`
I found the target `reviewable` depends on target `vet`, but simply running `make vet` it says 
> vet: pkg/configuration/container/container_kill_test.go:101:15: undefined: mocks.NewMockContainerAPIClient
which is generated in target `test-go-generate`.


### Update `EXCLUDES_DIRS` in `hack/license/header-check.sh`
I found one of the excluded directory is set to `pkg/lorry/component/`. But currently there is no directory named `component` under `pkg/lorry`. According to the failure cases below, seems to exclude `pkg/lorry/engines` and `pkg/lorry/util` is enough, but I am not sure if it is too arbitrary. @shanshanying 

Anyway I'd like to make a PR to fix these two issues first..

```bash
❯ make check-license-header |grep FAIL
Header check: pkg/lorry/engines/kafka/auth.go... FAIL
Header check: pkg/lorry/engines/kafka/consumer.go... FAIL
Header check: pkg/lorry/engines/kafka/kafka.go... FAIL
Header check: pkg/lorry/engines/kafka/metadata.go... FAIL
Header check: pkg/lorry/engines/kafka/metadata_test.go... FAIL
Header check: pkg/lorry/engines/kafka/producer.go... FAIL
Header check: pkg/lorry/engines/kafka/sarama_log_bridge.go... FAIL
Header check: pkg/lorry/engines/kafka/sasl_oauthbearer.go... FAIL
Header check: pkg/lorry/engines/kafka/scram_client.go... FAIL
Header check: pkg/lorry/engines/kafka/thirdparty/retry.go... FAIL
Header check: pkg/lorry/engines/kafka/utils.go... FAIL
Header check: pkg/lorry/engines/redis/metadata.go... FAIL
Header check: pkg/lorry/engines/redis/redis.go... FAIL
Header check: pkg/lorry/engines/redis/settings.go... FAIL
Header check: pkg/lorry/engines/redis/settings_test.go... FAIL
Header check: pkg/lorry/util/config/decode.go... FAIL
make: *** [check-license-header] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -162,7 +162,7 @@ fmt: ## Run go fmt against code.
 	$(GOFMT) -l -w -s $$(git ls-files --exclude-standard | grep "\.go$$" | grep -v $(GENERATED_CLIENT_PKG))
 
 .PHONY: vet
-vet: ## Run go vet against code.
+vet: test-go-generate ## Run go vet against code.
 	GOOS=$(GOOS) $(GO) vet -mod=mod ./...
 
 .PHONY: cue-fmt

--- a/hack/license/header-check.sh
+++ b/hack/license/header-check.sh
@@ -24,7 +24,7 @@ set -e -o pipefail
 # Initialize vars
 ERR=false
 FAIL=false
-EXCLUDES_DIRS="vendor/\|apis/\|tools/\|externalapis/\|pkg/lorry/component/\|pkg/cli/cmd/plugin/download\|pkg\/common"
+EXCLUDES_DIRS="vendor/\|apis/\|tools/\|externalapis/\|pkg/lorry/engines/\|pkg/lorry/util/\|pkg/cli/cmd/plugin/download\|pkg\/common"
 APACHE2_DIRS="apis/\|externalapis/"
 
 for file in $(git ls-files | grep '\.cue\|\.go$' | grep -v ${EXCLUDES_DIRS}); do

